atuin-client/src/database.rs

```diff
@@ -306,10 +306,7 @@ impl Database for Sqlite {
         }
 
         if unique {
```

Comment:
we probably want to reuse search at some point, rather than sorta duplicate it

the main difference is that search automatically de-dupes and has a few other options, while list does not.